lib/launcher.js

```diff
@@ -257,13 +257,23 @@ class Launcher {
             if (schedulableCaps.length === 0) {
                 break
             }
-
-            this.startInstance(
-                [schedulableCaps[0].specs.pop()],
-                schedulableCaps[0].caps,
-                schedulableCaps[0].cid,
-                schedulableCaps[0].seleniumServer
-            )
+            
+            if (config.runSpecInOrder) {
```

Comment:
Adding new configs makes setting up WebdriverIO just more complicated. We should be careful what we add to it.